tests/TestCase/Installer/PluginInstallerTest.php

```diff
@@ -166,6 +166,21 @@ public function testUpdateConfigAddPathFileExists()
     }
 
     /**
+     * testUpdateConfigAddRootPath
+     *
+     * @return void
+     */
+    public function testUpdateConfigAddRootPath() {
+        file_put_contents($this->path . '/config/plugins.php', '<?php return ["plugins" => ["Bake" => "/some/path"]];');
+
+        $this->installer->updateConfig('DebugKit', sys_get_temp_dir() . '/vendor/cakephp/debugkit');
+        $contents = file_get_contents($this->path . '/config/plugins.php');
+        $this->assertContains('<?php', $contents);
+        $this->assertContains("'DebugKit' => '/vendor/cakephp/debugkit/'", $contents);
```

Comment:
And where is the rest of the past completed? As in, where is the ROOT prepended when loading the plugin?